html-lists/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Lists</title>
+  </head>
+  <body>
+    <h1>HTML Lists</h1>
+    <h2>Ordered List Example</h2>
+    <ol>
+      <li>1. Learn to Code.</li>
+      <li>2. Build Projects.</li>
+      <li>3. Get a Job.</li>
```

Comment:
The 1, 2, 3 aren't needed. Since you're using an `ol`, it will already be numbered by default